### PR TITLE
Add itco watchdog to qemu_vm

### PR DESCRIPTION
**Pull request by yiqianwei**
qemu_vm.py:  add "-global ICH9-LPC.noreboot=off" cmd to guest cli

ID: 1462

### CODE REVIEW

**Comment by yiqianwei** [comment]
Test results: boot a guest with cmd
[stdlog]     -global ICH9-LPC.noreboot=off \
[stdlog]     -watchdog-action reset \

**Comment by yiqianwei** [comment]
@yanan-fu , Could you help to review this patch? thanks

**Comment by yanan-fu** [comment]
@yiqianwei  Could you add the issue ID in the patch description ? Thanks!

**Comment by yiqianwei** [comment]
> @yiqianwei Could you add the issue ID in the patch description ? Thanks!

@yanan-fu , I have updatedd it. please check it again,thanks

**Comment by yanan-fu** [comment]
LGTM

**Comment by yanan-fu** [comment]
@luckyh  Could you please help review this patch ? Thanks!

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -1439,13 +1439,15 @@ def add_sga(devices):
...
+            if device_type == "itco":
+                dev = qdevices.QGlobal('ICH9-LPC', 'noreboot', 'off')
+            elif device_type and devices.has_device(device_type):
                 if devices.is_pci_device(device_type):
                     parent_bus = self._get_pci_bus(self.params, None, False)
                 dev = QDevice(device_type, parent_bus=parent_bus)
                 set_cmdline_format_by_cfg(dev, self._get_cmdline_format_cfg(),
                                           "watchdog")
-                watchdog_devs.append(dev)
+            watchdog_devs.append(dev)
```
Comment (luckyh) [inline virttest/qemu_vm.py:1450]: Here the issue is that a `NameError` would be raised if both of the above `if`/`elif` clauses were bypassed somehow in this routine.

Comment (yanan-fu) [reply virttest/qemu_vm.py:1450]: Good catch !

Comment (yiqianwei) [reply virttest/qemu_vm.py]: Ok, Updated this issue, thanks

**Review by yanan-fu — APPROVED** [review]
Ack

**Comment by yanan-fu** [comment]
@luckyh  Could you help review the latest update ? Thanks!

**Review by luckyh — APPROVED** [review]
LGTM

**Comment by luckyh** [comment]
Thanks to all.

**Pull request merged**